web/pdf_viewer.css

```diff
@@ -46,6 +46,46 @@
   border: none;
 }
 
+.pdfViewer.scrollHorizontal, .pdfViewer.scrollWrapped, .spread {
+  text-align: center;
+}
+
+.pdfViewer.scrollHorizontal, .pdfViewer.scrollWrapped {
+  margin: 4px;
+}
+
+.pdfViewer.scrollHorizontal, .spread {
+  white-space: nowrap;
+}
+
+.pdfViewer.removePageBorders {
+  margin: 0;
+}
+
+.spread .page,
+.pdfViewer.scrollHorizontal .page,
+.pdfViewer.scrollWrapped .page,
+.pdfViewer.scrollHorizontal .spread,
+.pdfViewer.scrollWrapped .spread {
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.spread .page {
+  margin: 1px -3.5px -8px;
+}
+
+.pdfViewer.scrollHorizontal .page,
+.pdfViewer.scrollWrapped .page {
+  margin: -3px -3.5px -4px;
+}
+
+.pdfViewer.removePageBorders .spread .page,
+.pdfViewer.removePageBorders.scrollHorizontal .page,
+.pdfViewer.removePageBorders.scrollWrapped .page {
+  margin: 0 5px 10px;
```

Comment:
I am unable to figure out why these numbers have the values as shown here. Can you enlighten me?

Reply:
These are simply the values on line 36, with the horizontal margins changed from `auto` to 5px (the average of 0 and 10px) so that pages can sit next to each other in these modes.